Crashlytics/UnitTests/FIRCLSSettingsTests.m

```diff
@@ -387,7 +382,7 @@ - (void)testCorruptCache {
   // Cache them, and reload. Since it's corrupted we should delete it all
   [self cacheSettingsWithGoogleAppID:TestGoogleAppID
                     currentTimestamp:currentTimestamp
-                 expectedRemoveCount:2];
+                 expectedRemoveCount:3];
```

Comment:
Any reason we changed the expected count here without change any actual code logic?